<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetImportDialog</class>
 <widget class="QDialog" name="DatasetImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>387</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Load Dataset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="fileInfoGroupBox">
     <property name="title">
      <string>File Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="filenameLabel">
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="filenameLayout">
        <item>
         <widget class="QLineEdit" name="filenameLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fileSizeLabel_2">
        <property name="text">
         <string>File Size (MB)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="fileSizeLabel">
        <property name="text">
         <string>No File Selected</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="formatLabel">
        <property name="text">
         <string>Data Format</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="dataFormatComboBox">
        <item>
         <property name="text">
          <string>Wide Text</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Long Text</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wide Text (CSV)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Long Text (CSV)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vespucci Dataset</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Dataset Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="toolTip">
         <string>The dataset should have a unique name that is a valid filename on your operating system.</string>
        </property>
        <property name="maxLength">
         <number>32780</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="swapCheckBox">
        <property name="toolTip">
         <string>Depending on your settings, your data may have y in the column before x, rather than the traditional x before y.  Enabling this option will take the first column as y and second column as x.  Applies only to text files.</string>
        </property>
        <property name="text">
         <string>Swap x and y?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="spectraInfoGroupBox">
     <property name="title">
      <string>Spectra Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="yLabel">
        <property name="text">
         <string>Y-axis Description</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="yDescriptionLineEdit">
        <property name="text">
         <string>Intensity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>X-axis Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="xDescriptionLineEdit">
        <property name="text">
         <string>Raman Shift</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="yUnitsLabel">
        <property name="text">
         <string>Y-axis Units</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="xUnitsLabel">
        <property name="text">
         <string>X-axis Units</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="yUnitsLineEdit">
        <property name="text">
         <string>a.u.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="xUnitsLineEdit">
        <property name="text">
         <string>cm⁻¹</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
